libs/dh/metering-point/feature-overview/src/lib/primary-master-data/dh-metering-point-primary-master-data.component.spec.ts

```diff
@@ -25,14 +25,14 @@ import {
   queryByTerm,
   runOnPushChangeDetection,
 } from '@energinet-datahub/dh/shared/test-util-metering-point';
+import { MeteringPointType } from '@energinet-datahub/dh/shared/data-access-api';
```

Comment:
nitpick: sort imports

Reply:
Sorry it was on the wrong line, I think the emDash should be moved up to be sorted correctly :)

Reply:
like this:
`import { emDash } from '@energinet-datahub/dh/metering-point/shared/ui-util';
import { en as enTranslations } from '@energinet-datahub/dh/globalization/assets-localization';
import { getTranslocoTestingModule } from '@energinet-datahub/dh/shared/test-util-i18n';
import { MeteringPointType } from '@energinet-datahub/dh/shared/data-access-api';
import {
  getByTerm,
  getDefinitonByTerm,
  queryByTerm,
  runOnPushChangeDetection,
} from '@energinet-datahub/dh/shared/test-util-metering-point';`

Reply:
Do you mean sorting the imports alphabetically? Because the emDash is imported from a library and is in the same group of imports.